File: Survey Builder/Interface/ISurveyPinHandler.vb
Public Interface ISurveyPinHandler
    Function InsertPin(Pin As cSurveysPin) As Boolean
    Function GetSurveyPin(Pin As cSurveysPin) As cSurveysPin
    Function UpdatePinUsedGeneratedDateTime(Pin As cSurveysPin)
    Function DeleteSurveyPin(Pin As cSurveysPin)
End Interface

File: SurveyBuilderApp/SurveyBuilderApp/Interfaces/ISurveyPinHandler.cs
using SurveyBuilderApp.Classes;
using System;
using System.Collections.Generic;
using System.Text;
public interface ISurveyPinHandler
{
    string InsertPin(cSurveysPin Pin);
    string GetSurveyPin(cSurveysPin Pin);
    string UpdatePinUsedGeneratedDateTime(cSurveysPin Pin);
    string DeleteSurveyPin(cSurveysPin Pin);

}
